assets/package.json

```diff
@@ -41,8 +41,9 @@
   },
   "homepage": "https://github.com/usgs/waterdataui#readme",
   "devDependencies": {
-    "@fortawesome/fontawesome-free": "5.11.2",
     "autoprefixer": "^9.6.1",
+    "css-mqpacker": "^7.0.0",
+    "cssnano": "^4.1.10",
```

Comment:
Is this automatically being pulled in by postcss? The cssnano docs mention that it needs to be configured in a `postcss.config.js` file. :thinking:

Reply:
It is in postcss.config.js which was added to this repo.